frontend/src/packages/dashboard/components/SideBar/SideBar.tsx

```diff
@@ -113,6 +116,12 @@ export default function SideBar({
   isOpen,
   setOpen,
 }: Props) {
+
+    const folders = useSelector((state: RootState) =>
+      folderSelectors.getFolderState(state)
+    ).items;
+
+
```

Comment:
You can replace this code with the existing getFolderState() function in `/frontend/src/packages/dashboard/api/helpers.ts` !